htmlproofer/plugin.py

```diff
@@ -36,8 +37,7 @@ def on_post_page(self, output_content, config, **kwargs):
                 else:
                     print(error)
 
-
-
+    @lru_cache(maxsize = 500)
```

Comment:
It looks the LRU cache is set before the build of MkDocs config so the cache size cannot be passed in through config. @rhagenson, do you have any experience with this ?

Reply:
No, I do not have experience with this.

Reply:
okay, I'll leave it for future improvement